api/src/controllers/validation.ts

```diff
@@ -101,5 +102,35 @@ const beAValidUrl = {
     message: 'Must be a valid url',
 };
 
-export { beAResumeReviewState, beAValidUrl, beAValidUser, beAValidUuid, beProperlyUriEncoded };
+/**
+ * Test whether a field is properly Base64 encoded.
+ */
+const beProperlyBase64Encoded = {
+    predicate: async (field: string | undefined): Promise<boolean> => {
+        if (field === undefined || field === null) {
+            return false;
+        }
+        if (Buffer.from(field, 'base64').toString('base64') !== field) {
+            return false;
+        }
+        return true;
+    },
+    message: 'Must be properly base64 encoded',
+};
+
+/**
+ * Test whether a resume review already exists in the database.
+ */
+const beAValidResumeReview = {
+    predicate: async (field: string | undefined): Promise<boolean> => {
```

Comment:
Nit: since this uses the repository (a bit more specific than the other checks) should we rename the parameter to `resumeId`?

Reply:
Personally prefer `field` for consistency with other helpers and because we haven't yet verified that the value is in fact a valid resume id.

Reply:
Sounds good 👍